engine/src/history_table.rs

```diff
@@ -63,7 +98,11 @@ impl HistoryTable {
                 print!("|");
                 for file in 0..NumberOf::FILES {
                     let square = file + rank * NumberOf::FILES;
-                    print!("{:5} ", self.table[side as usize][piece_type][square]);
+                    let grid = self.table[side as usize][piece_type][square];
+                    print!(
+                        "{:5} | {:5}\n------\n[{:5} | {:5} ]",
+                        grid[0][0], grid[0][1], grid[1][0], grid[1][1]
+                    );
```

Comment:
Haven't actually tested this to see what it looks like - might have broken things.